packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -15,7 +15,7 @@ import {
 } from "@fluidframework/telemetry-utils/internal";
 import Deque from "double-ended-queue";
 
-import { InboundSequencedContainerRuntimeMessage } from "./messageTypes.js";
+import { type InboundSequencedContainerRuntimeMessageOrSystemMessage } from "./messageTypes.js";
```

Comment:
Let's figure out how to avoid  "Or System Message" leaking into here.  It's related to my other comment above about only calling RemoteMessageProcessor if `modernRuntimeMessage` is true.  Note that we only call into PSM if that's true as well.